app/src/main/java/com/example/dkirilova/gym/activities/MainActivity.java

```diff
@@ -113,63 +125,35 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
 
     public void openExerciseFragment() {
         configureToolbar(true, View.NO_ID);
-        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
-        ft.replace(R.id.fragmentContainer, new ExerciseFragment()).commit();
+        openFragment(new ExerciseFragment(), null, null);
     }
 
     public void openExerciseDetailsFragment(Gym gym) {
```

Comment:
leave only openFragment method and remove all  separate methods to open fragment